src/docs/asciidoc/faq.adoc

```diff
@@ -1,15 +1,43 @@
 [[faq]]
-= FAQ, best practices and other "How do I...?"
+= FAQ, Best Practices, and "How do I...?"
+
+[[faq.wrap-blocking]]
+== How do I wrap a synchronous, blocking call?
+
+It's often the case that a source of information is synchronous and blocking.
+To deal with such sources in your Reactor applications, apply the following
+pattern:
+
+[source,java]
+----
+Mono blockingWrapper = Mono.fromCallable(() -> { <1>
+    return /* make a remote synchronous call */ <2>
+});
+blockingWrapper.subscribeOn(Schedulers.elastic()); <3>
```

Comment:
this should be 
```
blockingWrapper = blockingWrapper.subscribeOn(Schedulers.elastic()); <3>
```

It is important to always affect the result of an operator to a variable, otherwise you'll lose the effect of that operator (this is actually covered by the faq item just below)